server/api/cvrs.py

```diff
@@ -915,11 +989,16 @@ def parse_cvr_ballots() -> Iterable[CvrBallot]:
                 )
 
             db_batch = batches_by_key.get(batch_number)
+            imprinted_id = (
+                cvr_guid_to_unique_identifier_mapping[cvr_guid]
+                if cvr_guid in cvr_guid_to_unique_identifier_mapping
+                else cvr_guid
+            )
```

Comment:
Should we throw an error if there's a CVR that's not in the ScannedBallotInformation file? Or do you think it's better to just fall back to the cvr guid?

Reply:
I went back and forth on this but landed on falling back to the CVR GUID as the safer option and the option least likely to require last-minute code changes on our end when Washington actually runs their audit next week

We can easily change this given what we learn from Washington's actual files and audit experience